From cefd57f76f7faec4143bf1e42ed148cad973c1f4 Mon Sep 17 00:00:00 2001
From: Fabrice Fontaine <fontaine.fabrice@gmail.com>
Date: Mon, 12 Aug 2019 23:17:21 +0200
Subject: [PATCH] configure.ac: fix so suffix

With uclibc or musl configuration, $host does not match to '*-linux-gnu*'.

Signed-off-by: Hiroshi Kawashima <kei-k@ca2.so-net.ne.jp>
[Retrieved (and slightly updated) from:
https://git.buildroot.net/buildroot/tree/package/gauche/0001-fix-so-suffix.patch]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
[Upstream status: https://github.com/shirok/Gauche/pull/508]
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 15092a2d6..b82b762a9 100644
--- a/configure.ac
+++ b/configure.ac
@@ -641,7 +641,7 @@ AS_CASE([$host],
     SHLIB_MAIN_LDFLAGS=""
     SHLIB_OK=ok
     ],
-  [*-linux-gnu*|*-*-gnu*|*freebsd*|*dragonfly*], [
+  [*-linux-*|*-*-gnu*|*freebsd*|*dragonfly*], [
     SHLIB_SO_CFLAGS="-fPIC"
     SHLIB_SO_LDFLAGS="$rpath -shared -o"
     SHLIB_SO_SUFFIX="so"
-- 
2.20.1

